From 863e7dd43df2925705003424ba52806f6a3da001 Mon Sep 17 00:00:00 2001
From: Cary Phillips <cary@ilm.com>
Date: Wed, 14 Feb 2024 10:38:20 -0800
Subject: [PATCH] Add std:: to isfinite in ImathFun.cpp (#368)

Signed-off-by: Cary Phillips <cary@ilm.com>

ImathFun.cpp: add std:: to isfinite in remaining cases (#371)

A follow-up to https://github.com/AcademySoftwareFoundation/Imath/pull/368
Fixes: https://github.com/AcademySoftwareFoundation/Imath/issues/367

Signed-off-by: Sergey Fedorov <vital.had@gmail.com>
---
 src/Imath/ImathFun.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git src/Imath/ImathFun.cpp src/Imath/ImathFun.cpp
index bfec292..c8c2e7e 100644
--- src/Imath/ImathFun.cpp
+++ src/Imath/ImathFun.cpp
@@ -10,25 +10,25 @@ IMATH_INTERNAL_NAMESPACE_SOURCE_ENTER
 
 float succf(float f) IMATH_NOEXCEPT
 {
-    return isfinite(f) ?
+    return std::isfinite(f) ?
             std::nextafter(f, std::numeric_limits<float>::infinity()) : f;
 }
 
 float predf(float f) IMATH_NOEXCEPT
 {
-    return isfinite(f) ?
+    return std::isfinite(f) ?
             std::nextafter(f, -std::numeric_limits<float>::infinity()) : f;
 }
 
 double succd(double d) IMATH_NOEXCEPT
 {
-    return isfinite(d) ?
+    return std::isfinite(d) ?
             std::nextafter(d, std::numeric_limits<double>::infinity()) : d;
 }
 
 double predd(double d) IMATH_NOEXCEPT
 {
-    return isfinite(d) ?
+    return  std::isfinite(d) ?
             std::nextafter(d, -std::numeric_limits<double>::infinity()) : d;
 }
 
